src/Persistence.LinnApps/ServiceDbContext.cs

```diff
@@ -949,5 +939,43 @@ private static void BuildReqSerialNumbers(ModelBuilder builder)
             r.Property(l => l.Sequence).HasColumnName("SEQ");
             r.Property(l => l.SerialNumber).HasColumnName("SERNOS_NUMBER");
         }
+
+        private static void BuildStoresPallets(ModelBuilder builder)
+        {
+            var r = builder.Entity<StoresPallet>().ToTable("STORES_PALLETS");
+            r.HasKey(l => l.PalletNumber);
+            r.Property(l => l.PalletNumber).HasColumnName("PALLET_NUMBER");
+            r.Property(l => l.Description).HasColumnName("DESCRIPTION").HasMaxLength(50);
+            r.Property(l => l.DateInvalid).HasColumnName("DATE_INVALID");
+            r.Property(l => l.LocationIdCode).HasColumnName("LOCATION_ID");
+            r.Property(l => l.DateLastAudited).HasColumnName("DATE_LAST_AUDITED");
+            r.Property(l => l.Accessible).HasColumnName("ACCESSIBLE").HasMaxLength(1);
+            r.Property(l => l.StoresKittable).HasColumnName("STORES_KITTABLE").HasMaxLength(1);
+            r.Property(l => l.StoresKittablePriority).HasColumnName("STORES_KITTABLE_PRIORITY");
+            r.Property(l => l.SalesKittable).HasColumnName("SALES_KITTABLE").HasMaxLength(1);
+            r.Property(l => l.SalesKittablePriority).HasColumnName("SALES_KITTABLE_PRIORITY");
+            r.Property(l => l.AllocQueueTime).HasColumnName("ALLOC_QUEUE_TIME");
+            r.Property(l => l.AuditedBy).HasColumnName("AUDITED_BY");
+            r.Property(l => l.LocationTypeId).HasColumnName("LOCATION_TYPE").HasMaxLength(1);
+            r.Property(l => l.DefaultStockPoolId).HasColumnName("DEFAULT_STOCK_POOL").HasMaxLength(10);
+            r.Property(l => l.StockType).HasColumnName("TYPE_OF_STOCK").HasMaxLength(1);
+            r.Property(l => l.StockState).HasColumnName("STOCK_STATE").HasMaxLength(1);
+            r.Property(l => l.AuditOwnerId).HasColumnName("AUDIT_OWNER_ID");
+            r.Property(l => l.AuditFrequencyWeeks).HasColumnName("AUDIT_FREQUENCY_WEEKS");
+            r.Property(l => l.AuditedByDepartmentCode).HasColumnName("AUDITED_BY_DEPARTMENT_CODE").HasMaxLength(10);
+            r.Property(l => l.MixStates).HasColumnName("MIX_STATES").HasMaxLength(1);
+            r.Property(l => l.Cage).HasColumnName("CAGE").HasMaxLength(1);
+            r.HasOne(l => l.LocationId).WithMany().HasForeignKey(p => p.LocationIdCode);
+            r.HasOne(l => l.LocationType).WithMany().HasForeignKey(p => p.LocationTypeId);
```

Comment:
if you delete the explicit id types on the entity you can keep it working (hopefully) by changing these sorts of mappings to just refernce th db foreign key directly, i.e. 

r.HasOne(l => l.LocationType).WithMany().HasForeignKey(p => "LOCATION_TYPE_ID"; (or whatever the name of the FK is)